src/types/onyx/ReportAction.ts

```diff
@@ -52,6 +52,9 @@ type ReportActionBase = {
     /** The ID of the previous reportAction on the report. It is a string represenation of a 64-bit integer (or null for CREATED actions). */
     previousReportActionID?: string;
 
+    /** ID of a report */
```

Comment:
Can this comment be more specific like 
```suggestion
    /** The ID of the report which the action belongs to */
```

Reply:
Updated